src/drive/web/modules/move/MoveModal.jsx

```diff
@@ -71,30 +91,44 @@ class MoveModal extends React.Component {
   }
 
   cancelMove = async entries => {
-    const { t } = this.context
+    const { client, t } = this.context
 
     try {
       await Promise.all(
-        entries.map(entry => this.moveEntry(entry._id, entry.dir_id))
+        entries.map(entry =>
+          CozyFile.move(entry._id, { folderId: entry.dir_id })
+        )
       )
+      const fileCollection = client.collection(CozyFile.doctype)
+      let restoreErrorsCount = 0
+      await Promise.all(
+        this.state.trashedFiles.map(async id => {
+          try {
+            await fileCollection.restore(id)
+          } catch (e) {
+            restoreErrorsCount++
+          }
+        })
+      )
+
       Alerter.info(
         t('Move.cancelled', {
           subject: entries.length === 1 ? entries[0].name : '',
           smart_count: entries.length
         })
       )
+
+      if (restoreErrorsCount) {
+        Alerter.error(
```

Comment:
Doesn't that lead to "flashing" alert messages, that cover each other up, if there are both sucesses and errors?

Reply:
:+1: Changed!